ocaml-lsp-server/test/e2e/__tests__/textDocument-codeAction.test.ts

```diff
@@ -601,4 +619,262 @@ let x = _
       ]
     `);
   });
+
+  it("", async () => {
```

Comment:
it would be nice to have names for tests; here it could be "add missing rec in toplevel let-binding"